client/components/editor/index.vue

```diff
@@ -40,16 +41,18 @@ export default {
     repositoryId: { type: Number, required: true },
     activityId: { type: Number, required: true }
   },
-  data: () => ({
-    isLoading: true,
-    selectedElement: null
-  }),
+  data: () => ({ isLoading: true }),
   computed: {
     ...mapGetters('editor', ['activity', 'contentContainers', 'rootContainerGroups']),
     ...mapGetters('repository', ['repository', 'outlineActivities']),
     ...mapGetters('repository/userTracking', ['getActiveUsers']),
+    ...mapGetters('repository/contentElements', ['elements']),
     ...mapState('editor', ['showPublishDiff']),
-    activeUsers: vm => vm.getActiveUsers('activity', vm.activityId)
+    activeUsers: vm => vm.getActiveUsers('activity', vm.activityId),
+    selectedElement() {
+      const { elementId } = this.$route.query;
+      return find(this.elements, { uid: elementId });
+    }
```

Comment:
Does it make sense to move this to the `repository/contentElements` getter so it can be easily reused?